pandas/io/formats/style.py

```diff
@@ -2911,7 +2922,13 @@ def css_calc(x, left: float, right: float, align: str):
             else:
                 start, end = z_frac, (x - left) / (right - left)
 
-        return css_bar(start * width, end * width, color)
+        ret = css_bar(start * width, end * width, color)
```

Comment:
what happens for height / width out of bounds? do we raise, is this tested?

Reply:
currently still renders, but will get bad results. `height` is a css artifact though so bad CSS just means it gets ignored by the browser.

What do you want to do? Leave as is? Raise on out-of-bounds? or auto trim bad values to be in the range?

Reply:
i think raising is appropriate (don't have to do here), prob needs a dedicated PR as likely need to audit all of these.